ext/nnstreamer/tensor_filter/tensor_filter_tvm.cc

```diff
@@ -320,10 +320,13 @@ tvm_subplugin::invoke (const GstTensorMemory *input, GstTensorMemory *output)
   tvm::runtime::NDArray tensor;
 
   /* read functions */
-  tvm::runtime::PackedFunc set_input = gmod.GetFunction ("set_input");
+  tvm::runtime::PackedFunc set_input = gmod.GetFunction ("set_input_zero_copy");
   if (set_input == nullptr) {
-    cleanup ();
-    throw std::runtime_error ("Packed function `set_input` not defined in model");
+    set_input = gmod.GetFunction ("set_input");
+    if (set_input == nullptr) {
+      cleanup ();
+      throw std::runtime_error ("Packed function `set_input` not defined in model");
```

Comment:
What if the pipeline is:
``` SOMESRC ! tensor_filter framework=tvm model=a ! ... ```
and ```SOMESRC``` is a third party element that we do not know if it is aligned or not?

Reply:
Your code needs to be ready for such situation and you need test cases for such cases as well.

Reply:
Is it ok to handle it by try & catch?
If not aligned then zero-copy throws an error so that I can catch it and retry with copying input